src/routes/register/components/memberInfoForm.svelte

```diff
@@ -3,47 +3,166 @@
     import ContinueButton from "./continueButton.svelte";
     import BackButton from "./backButton.svelte";
 
+    let invalidInputs = false;
+
+    const indicatorStatus = {
+        none: "",
+        error: "!border-error-600",
+    };
+
+    // Will get value according to status of validator
+    let birthDateIndicator = indicatorStatus.none;
+
     // Indicates how many steps of the register process have been completed
     export let currentStep = 0;
+
+    // Formfield binding references
+    export let formData = {
+        firstName,
+        lastName,
+        street,
+        houseNumber,
+        postalCode,
+        city,
+        country,
+        birthDate,
+        birthPlace,
+        phoneNumber,
+    };
+
+    // Functions
+    function isAtLeast18YearsOld(birthDateString) {
+        const birthDate = new Date(birthDateString);
+        const today = new Date();
+
+        // Calculate age
+        let age = today.getFullYear() - birthDate.getFullYear();
+
+        // Adjust age if the birthdate hasn't occured yet this year
+        const birthDateThisYear = new Date(
+            today.getFullYear(),
+            birthDate.getMonth(),
+            birthDate.getDate(),
+        );
+        if (today < birthDateThisYear) {
+            age--;
+        }
+
+        return age >= 18;
+    }
+
+    function validateInput() {
+        invalidInputs = false;
+
+        if (!isAtLeast18YearsOld(formData.birthDate)) {
+            invalidInputs = true;
+            birthDateIndicator = indicatorStatus.error;
+            return;
+        } else {
+            birthDateIndicator = indicatorStatus.none;
+        }
```

Comment:
```suggestion
        }
        
        birthDateIndicator = indicatorStatus.none;
```
Im `if` ist ein `return`, also brauchst du kein `else` hier.

Reply:
Ja, das ist tatsächlich schöner so.